browser/app/js/browser/Login.js

```diff
@@ -16,24 +16,22 @@
 
 import React from "react"
 import { connect } from "react-redux"
-import classNames from "classnames"
 import logo from "../../img/logo.svg"
 import Alert from "../alert/Alert"
 import * as actionsAlert from "../alert/actions"
 import InputGroup from "./InputGroup"
 import web from "../web"
 import { Redirect, Link } from "react-router-dom"
-import qs from "query-string"
-import storage from "local-storage-fallback"
-import history from "../history"
+import { OpenIDLoginButton } from './OpenIDLoginButton'
```

Comment:
The PR is looking good to me. However a very small nitpick here, we don't need the curly braces around OpenIDLoginButton as it is already a default export.